### ISSUE TITLE
Particle.js Module parse failed: Unexpected token (13:5)

### ISSUE THREAD

**Issue by yusufdeepwork** [drawcall/Proton#78]
```
ERROR in ./node_modules/proton-engine/src/behaviour/Behaviour.js 6:14
Module parse failed: Unexpected token (6:14)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| 
,```

``
![resim](https://user-images.githubusercontent.com/57027351/96580601-34572500-12e1-11eb-9fd1-3ade6577dd3b.png)


I take error . I can't use this . I am so sorry.

**Comment by fantasticsoul** [comment]
Can u put a minimal project to git that can reproduce this bug?

**Comment by yusufdeepwork** [comment]
> Can u put a minimal project to git that can reproduce this bug?

can i add you in my private repository for showing ?

**Comment by yusufdeepwork** [comment]
> Can u put a minimal project to git that can reproduce this bug?

https://github.com/yusufdeepwork/Candidate-Student-Application I have opened for everyone.

**Comment by yusufdeepwork** [comment]
I have come back proton-engine@4.1.5 and it works.
I guess Latest proton-engine version has a problem.
`npm i proton-engine@4.1.5`

**Comment by bohrzhang** [comment]
remove the module field from the package.json file and republish will work well.

**Comment by drawcall** [comment]
> I have come back proton-engine@4.1.5 and it works.
> I guess Latest proton-engine version has a problem.
> `npm i proton-engine@4.1.5`


Oh, indeed, as you said, there will be some problems when webpack loads. I have solved it and upgraded to v4.2.1
